frontend/src/AlertsManagement/AlertsContent.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Box from '@material-ui/core/Box';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import AllInboxIcon from '@material-ui/icons/AllInbox';
+import DoneIcon from '@material-ui/icons/Done';
+import ErrorIcon from '@material-ui/icons/Error';
+import AlertsList from './AlertsList';
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      aria-labelledby={`tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+};
+
+let a11yProps = (index) => {
+  return {
+    id: `tab-${index}`,
+    'aria-controls': `tabpanel-${index}`,
+  };
+}
+
+// TODO: fetch alerts from backend and save in arrays.
+const allAlerts = [0, 1, 2, 3, 4, 5, 6];
+const unresolvedAlerts = [0, 1, 2, 3];
+const resolvedAlerts = [4, 5, 6]
+
+class AlertsContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tab: 0,
```

Comment:
is there a 1-1 mapping between the tab index and the naming (ie will tab '0' always be unresolved, tab (1?i assume) will be resolved, and tab 2 will be 'all'?

could we make the tab state some enum instead of indices? i'm used to typescript with native enum, but not sure how clean that will translate to js.

the purpose is when you change state/set state, you'll know exactly what the tab represents.
ie tab: UNRESOLVED, or tab: RESOLVED, etc instead of using these 'semi-magic' numbers.

Reply:
Yes, there's a 1-1 mapping! Made them enums